scripts/lint_prowjobs/main.go

```diff
@@ -191,7 +191,7 @@ func getFilesChanged(gitRoot string, pullBaseSha string, pullPullSha string) ([]
 
 	filesChanged := strings.Fields(string(gitDiffOutput))
 	for _, file := range filesChanged {
-		if strings.Contains(file, "presubmits") {
+		if strings.Contains(file, "presubmits") && strings.HasPrefix(file, "jobs") && strings.HasSuffix(file, "yaml") {
```

Comment:
@markapruett yes, this has been added to check prowjobs only in the jobs folder. before this the presubmit linter were failing because it checked any files with "presubmits" in it and thus cannot unmarshal minimal configs in the templater.